packages/context/src/binding-decorator.ts

```diff
@@ -110,3 +113,18 @@ export namespace bind {
     };
   }
 }
+
+/**
+ * Alias decorator `@injectable` to `@injectable` for backward compatibility
+ * @param specs Binding specs
+ */
```

Comment:
Should we add `@deprecated` keyword and point users to the new `@injectable` decorator?

Reply:
See https://devblogs.microsoft.com/typescript/announcing-typescript-4-0/#deprecated-support:

<img width="776" alt="Screen Shot 2020-09-10 at 16 15 27" src="https://user-images.githubusercontent.com/1140553/92743485-de599e00-f380-11ea-9758-24364dd964d3.png">

Reply:
I'm reluctant to mark `@bind` deprecated now as it has been widely used in existing applications.  I would rather treat it as an alias to `@injectable`.

Reply:
Agreed, maybe too early to mark `@bind` as deprecated.